api/v1/ytsaurus_webhook.go

```diff
@@ -451,63 +460,102 @@ func (r *Ytsaurus) validateInstanceSpec(instanceSpec InstanceSpec, path *field.P
 	return allErrors
 }
 
-func (r *Ytsaurus) validateYtsaurus(old *Ytsaurus) field.ErrorList {
+func (r *ytsaurusValidator) validateExistsYtsaurus(ctx context.Context, newYtsaurus *Ytsaurus) field.ErrorList {
+	var allErrors field.ErrorList
+
+	var ytsaurusList YtsaurusList
+
+	err := r.Client.List(ctx, &ytsaurusList, &client.ListOptions{Namespace: newYtsaurus.Namespace})
+	ytsauruslog.Info("validateExistsYTsaurus", "ytsaurusList", ytsaurusList)
+
+	if err != nil && !apierrors.IsNotFound(err) {
+		allErrors = append(allErrors, field.InternalError(field.NewPath("k8sClient"), err))
+		return allErrors
+	}
+
+	for _, ytsaurus := range ytsaurusList.Items {
+		if ytsaurus.Name == newYtsaurus.Name {
```

Comment:
So. Validation will accept any update when namespace already has more than one ytsaurus instance.
This is fine, but we could do better.
For example: accept only updates to new spec with isManaged = false.

Reply:
Improved logic in that place, please take a look https://github.com/ytsaurus/ytsaurus-k8s-operator/pull/305/commits/c328594faf54c70202eb093de5caf7dbad5793d3